Source/WarpX.cpp

```diff
@@ -692,6 +692,24 @@ WarpX::ReadParameters ()
 
         // Only needs to be set with WARPX_DIM_RZ, otherwise defaults to 1.
         pp.query("n_rz_azimuthal_modes", n_rz_azimuthal_modes);
+#if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
+        // Force use of cell centered in r and z.
+        // Also, do_nodal is forced to be true. Here, do_nodal effectively
+        // means do_colocated (i.e. not staggered).
+        do_nodal = true;
```

Comment:
Instead of overriding the user's input, could we use an `AMREX_ALWAYS_ASSERT` here, in order to prompt the user to set `do_nodal=true` in their input script?

Reply:
I think it would be an unnecessary complication for the user to have to set do_nodal=true, especially since its use is somewhat confusing since the solver is actually cell centered and not nodal.